### PR TITLE
Support inline rendering

### PR DESCRIPTION
**Pull request by IlyaSemenov**
This adds support for inline rendering as requested in #4.

I merged all entry accessors (HTML output, rel_urls, inline HTML output and raw content) within a single object. The old `rel_js` and `rel_css` are still supported for compatibility reasons.

Frankly the whole refactoring is somewhat of overkill, but I just couldn't stop enjoying finishing this up :)

### CODE REVIEW

**Comment by markfinger** [comment]
Cool. Thanks for all the work.

I'm wont have time to dig through the changes until early next week. Nonetheless, I did a superficial pass and everything looks good

**Comment by IlyaSemenov** [comment]
No worries, I added a manual workaround to my project for the time being anyway.

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Usage
 If you installed with pip, import it with
 
 ```python
-import webpack_manifest from webpack_manifest
+from webpack_manifest import webpack_manifest
```
Comment (markfinger) [inline README.md:42]: Thanks for the catch. I'd probably been in JS land when I wrote the docs :)

**Comment by markfinger** [comment]
Cool. Looks great.

Thanks again for the changes!

**Comment by markfinger** [comment]
`1.2.0` has been published to PyPI with your changes

**Pull request merged**